### PR TITLE
add get drone flight mode method

### PR DESCRIPTION
**Pull request by ashum68**

### CODE REVIEW

File: mavlink/modules/flight_controller.py [hunk 1]
```diff
@@ -235,3 +235,12 @@ def set_flight_mode(self, mode: str) -> bool:
             print("ERROR: an unsupported flight mode is set by dronekit.VehicleMode()")
             return False
         return True
+
+    def get_flight_mode(self) -> "tuple[bool, str | None]":
```
Comment (Xierumeng) [inline mavlink/modules/flight_controller.py:239]: Return an enumeration instead of string.

**Review by Xierumeng — COMMENTED** [review]
Reviewed.

File: mavlink/modules/flight_controller.py (hunk 1 above)
Comment (ashum68) [reply mavlink/modules/flight_controller.py]: see changes

**Review by TongguangZhang — APPROVED** [review]
lgtm!

**Pull request merged**